tests/test_sshd.test

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env bash
-
+# Copyright (C) The libssh2 project and its contributors.
+# SPDX-License-Identifier: BSD-3-Clause
+#
 # Written by Simon Josefsson and Viktor Szakats
```

Comment:
Same case as `Makefile.mk`: Maybe we could convert the "written by" line to a copyright header here? It means the same thing in practice, though IANAL.

Reply:
:+1: